MediaBrowser.Api/Playback/Progressive/BaseProgressiveStreamingService.cs

```diff
@@ -281,10 +281,9 @@ protected async Task<object> ProcessRequest(StreamRequest request, bool isHeadRe
         /// <returns>Task{System.Object}.</returns>
         private async Task<object> GetStaticRemoteStreamResult(StreamState state, Dictionary<string, string> responseHeaders, bool isHeadRequest, CancellationTokenSource cancellationTokenSource)
         {
-            string useragent = null;
-            state.RemoteHttpHeaders.TryGetValue("User-Agent", out useragent);
+            state.RemoteHttpHeaders.TryGetValue(HeaderNames.UserAgent, out var useragent);
 
-            var trySupportSeek = false;
+            const bool trySupportSeek = false;
```

Comment:
Consts should be PascalCase

Reply:
Not configured in the editorconfig

Reply:
Rider is basing that on local consts currently in the codebase, right?
I'd have to look, but I'm pretty sure analyzers want PascalCase.